lib/stellar_core_commander/local_process.rb

```diff
@@ -124,15 +129,15 @@ def crash
       `kill -ABRT #{@pid}`
     end
 
-    private
```

Comment:
Please don't change encapsulation by simply removing the private keyword.    launch_stellar_core should remain private.